test/text/numeric_test.cc

```diff
@@ -670,12 +670,12 @@ TEST(TextNumericTest, FloatToStr_f64) {
       {"-1234.5", Base::Decimal, 0xc0934a00'00000000ull},
       {"15", Base::Decimal, 0x402e0000'00000000ull},
       {"-15", Base::Decimal, 0xc02e0000'00000000ull},
-      {"5e-324", Base::Decimal, 0x00000000'00000001ull},
-      {"-5e-324", Base::Decimal, 0x80000000'00000001ull},
+      {"4.9406564584124654e-324", Base::Decimal, 0x00000000'00000001ull},
+      {"-4.9406564584124654e-324", Base::Decimal, 0x80000000'00000001ull},
       {"2.2250738585072014e-308", Base::Decimal, 0x00100000'00000000ull},
       {"-2.2250738585072014e-308", Base::Decimal, 0x80100000'00000000ull},
-      {"2.225073858507201e-308", Base::Decimal, 0x000fffff'ffffffffull},
-      {"-2.225073858507201e-308", Base::Decimal, 0x800fffff'ffffffffull},
+      {"2.2250738585072009e-308", Base::Decimal, 0x000fffff'ffffffffull},
+      {"-2.2250738585072009e-308", Base::Decimal, 0x800fffff'ffffffffull},
```

Comment:
Whats up with these diffs?

Reply:
I'm getting an additional digit of precision, though I'm not quite sure why. ISTR that you need 17 digits of precision for `double`, and 9 digits for `float` for round-tripping. I suppose the gdtoa library is determining that "...01" is sufficient to unambiguously specify the number, but I'm not sure why absl's formatter isn't.